templates/hubnetworking/terraform.tf

```diff
@@ -1,11 +1,21 @@
 terraform {
   required_version = "~> 1.6"
   required_providers {
-    azurerm = "~> 3.88"
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.107"
+    }
+    azapi = {
+      source  = "Azure/azapi"
+      version = "~> 1.13"
+    }
   }
   # backend "azurerm" {}
 }
 
+provider "azapi" {
```

Comment:
@matt-FFFFFF Should this provider block have `subscription_id = var.subscription_id_management`? I'm going to test this with a customer that is getting auth issues, but wondering if you already hit this issue?

Reply:
Hey - it needs rights to deploy MG diag settings plus the AMA stuff

Reply:
Therefore the higher level assignment at MG scope should be sufficient for this, depending on the order of operations in moving the sub into the MG

Reply:
I don't think the change made any difference to the customer anyway. I suspect something else is going on in their environment. I have tested with this change though and think it is valid to leave it like this for the accelerator.